modules/tensor_mechanics/include/interfacekernels/CZMInterfaceKernel.h

```diff
@@ -0,0 +1,46 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "InterfaceKernel.h"
+
+/// Forward Declarations
+class CZMInterfaceKernel;
+
+template <>
+InputParameters validParams<CZMInterfaceKernel>();
+
+/// DG kernel implementing CZM for a 1D/2D/3D traction sepration law based on
```

Comment:
Think about the person who stumbles on this and has no idea what CZM means -- best to spell that acronym out.

Reply:
ok done